src/main/java/me/partlysanestudios/partlysaneskies/mixin/MixinDungeonGuideCollectDiagnostics.java

```diff
@@ -19,7 +19,11 @@
 public class MixinDungeonGuideCollectDiagnostics {
     @Inject(method = "sendLogActually*", at = @At("HEAD"), cancellable = true)
     public void onSendLogActuallyHead(CallbackInfo ci) {
-        if (PartlySaneSkies.config.blockDgDiagnostics)
+        if (PartlySaneSkies.config == null) {
```

Comment:
(!PartlySaneSkies.config)

Reply:
That's not valid syntax (this isn't js theres no typecohersion)

Reply:
Right